sentry-ruby/lib/sentry/utils/logging_helper.rb

```diff
@@ -1,7 +1,12 @@
 module Sentry
   module LoggingHelper
-    def log_error(message)
-      logger.error(LOGGER_PROGNAME) { message }
+    def log_error(message, exception, debug: false)
```

Comment:
Was this considered public API? If so, the new required argument would probably be a breaking change.

Reply:
this is not meant to be a public API and actually doesn't exist in previous versions. it's introduced in a previous refacotring.